securesystemslib/keys.py

```diff
@@ -1720,14 +1403,15 @@ def decrypt_key(encrypted_key, passphrase):
 def create_rsa_encrypted_pem(private_key, passphrase):
   """
   <Purpose>
-  Return a string in PEM format, where the private part of the RSA key is
-  encrypted. The private part of the RSA key is encrypted by the Triple
-  Data Encryption Algorithm (3DES) and Cipher-block chaining (CBC) for the
-  mode of operation. Password-Based Key Derivation Function 1 (PBKF1) + MD5
-  is used to strengthen 'passphrase'.
+    Return a string in PEM format, where the private part of the RSA key is
+    encrypted. The private part of the RSA key is encrypted with
+    AES-256-CTR-Mode, and passwords are strengthened with PBKDF2-HMAC-SHA256
+    (100K iterations by default, but may be overriden in
+    'securesystemslib.settings.PBKDF2_ITERATIONS' by the user).
```

Comment:
Is this docstring accurate? It looks more like this function should call `pyca_crypto_keys.encrypt_key`, instead of `pyca_crypto_keys.create_rsa_encrypted_pem`?